Events/Controllers/EventController.cs

```diff
@@ -29,7 +29,130 @@ public EventController()
        
 
 
-       
+        public async Task<List<Event>> RetrieveEvents()
+        {
+            List<Event> events = new List<Event>();
+         
+            var request = new ScanRequest
+            {
+                TableName = "Events"
+            };
+            var response = await _dynamoDBClient.ScanAsync(request);

```

Comment:
It is better to isolate the persistence logic in a separate layer (repository class), read about the Repository pattern and also the vertical slicing pattern it will be good to know about them.